charts/zac/templates/secret.yaml

```diff
@@ -14,6 +14,7 @@ stringData:
   KVK_API_KEY: "{{ required "Valid .Values.kvkApi.apiKey entry required!" .Values.kvkApi.apiKey | toString }}"
   OBJECTS_API_TOKEN: "{{ required "Valid .Values.objectenApi.token entry required!" .Values.objectenApi.token | toString }}"
   OPEN_NOTIFICATIONS_API_SECRET_KEY: "{{ required "Valid .Values.notificationsSecretKey entry required!" .Values.notificationsSecretKey | toString }}"
+  PABC_API_KEY: "{{ required "Valid .Values.pabcApi.apiKey entry required!" .Values.pabcApi.apiKey | toString }}"
```

Comment:
This one should not be required? Because we don't have this PABC key for all official PodiumD environments (like Groningen). If you make this required we cannot deploy ZAC to Groningen etc anymore I think?!

Reply:
All PABC stuff must be optional for now (until we are finished with IAM phase 1).